gremlin-dotnet/src/Gremlin.Net/Process/Traversal/GraphTraversal.cs

```diff
@@ -177,6 +177,15 @@ private GraphTraversal(ICollection<ITraversalStrategy> traversalStrategies, Byte
             return Wrap<TStart, TEnd>(this);
         }
 
+        /// <summary>
+        ///     Adds the all step to this <see cref="GraphTraversal{SType, EType}" />.
+        /// </summary>
+        public GraphTraversal<TStart, TEnd> All (P? predicate)
```

Comment:
can `predicate` be null?

Reply:
Yes it can. Good catch. Updated implementation and tests to handle this.